tests/helpers/calc/test_expressions.py

```diff
@@ -189,3 +194,11 @@ def test_div_by_zero():
         evaluator("1/0")
     with raises(CalcZeroDivisionError, match=msg):
         evaluator("0^-1")
+
+def test_suffix_capitalization_error():
+    variables = {}
+    functions = {}
+    suffixes = {'M'}
+    match = "Invalid Input: m not permitted directly after a number. \(did you mean M\?\)"
```

Comment:
Nit: "D" in did again.

Reply:
Actually, keeping inline with other things, maybe just drop the period after "number".